meson_cpu/main_config.h.in

```diff
@@ -11,7 +11,7 @@
  */
 #ifndef @P@_CPU_DISPATCHER_CONF_H_
 #define @P@_CPU_DISPATCHER_CONF_H_
-/// This definition is required to provides comptablity with NumPy distutils
+/// This definition is required to provide comptibility with NumPy distutils
```

Comment:
Hmm, I think this is still wrong. Missed it.

Reply:
Ooops... I will run codespell again at some point, it should catch this typo.